libr/bin/p/bin_elf.inc

```diff
@@ -142,23 +142,12 @@ static RBinAddr* binsym(RBinFile *bf, int sym) {
 }
 
 static bool is_wordable_section(const char *name) {
-	if (!strcmp (name, ".init_array")) {
-		return true;
-	}
-	if (!strcmp (name, ".fini_array")) {
-		return true;
-	}
-	if (!strcmp (name, ".data.rel.ro")) {
-		return true;
-	}
-	if (!strcmp (name, ".dynamic")) {
-		return true;
-	}
-	if (!strcmp (name, ".got")) {
-		return true;
-	}
-	if (strstr (name, ".rela.")) {
```

Comment:
.rela. is strstr not strcmp. can you fix this?

Reply:
Should be fixed on https://github.com/radareorg/radare2/pull/21295/commits/a3427b36ad123b58863e708d2e6f45465b337453